playbooks/ci-ceph_swift/templates/usr/bin/lshw

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+hostname="{{ inventory_hostname }}"
+
+{% raw -%}
+# run the original lshw command
+if [ "$1" != "-C" -o "$2" != "disk" ]; then
+  if [ -f /usr/bin/lshw ]; then
+    sudo /usr/bin/lshw
+  fi
+fi
+
+# Output additional content for disks
+arr_label=(`lsblk -db |awk '{if(NR!=1){print $1}}'`)
+arr_size=(`lsblk -db |awk '{if(NR!=1){print $4/1024/1024/1024}}'`)
```

Comment:
1. No need for the conditional... `NR>1 {print}`
2. 1024/1024/1024 should just be a variable defined as the size of 1GiB for clarity... e.g. 
```
GB=1073741824
lsblk -db |awk -v gb=$GB 'NR>1{print $4/gb}'
```